### ISSUE TITLE
Valid and invalid example components don't meet WCAG AA level

### ISSUE THREAD

**Issue by juliavs85** [moodle/devdocs#779]
### What is the issue that you're experiencing?

The validexample and invalidexample components fail the WCAG AA level because of the lack of colour contrast of the text.
See https://colourcontrast.cc/06a400/ffffff and  https://colourcontrast.cc/fa383e/ffffff
Changing the text to black fixes the issues. 

### Are you able to provide a patch for this?

None
